artifacts/examples/getchangedblock_example.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: differentialsnapshot.k8s.io/v1alpha1
```

Comment:
change the suffix to "example.com"

Reply:
I think that we can fake it with the annotation that I added  https://github.com/phuongatemc/diffsnapcontroller/pull/1/files#diff-46c1810c6aa463d8678ee8338e8c7e8ae85bfc361ab886e4b1ab86c374485cbfR8

Reply:
To save time, let merge it and we will adjust it later.  This is just a sample anyway.